app/src/common/utils/timeDateUtils.test.js

```diff
@@ -0,0 +1,91 @@
+import { getDuration, approximateTimeFormat, dateFormat, daysBetween } from './timeDateUtils';
+
+const NOW = Date.now();
+const ONE_SECOND = 1000;
+const ONE_MINUTE = 60 * 1000;
+const ONE_HOUR = 3600 * 1000;
+const ONE_DAY = 24 * 3600 * 1000;
+
+describe('getDuration', () => {
+  test('should return sting', () => {
+    expect(typeof getDuration(NOW, NOW + ONE_HOUR)).toBe('string');
+  });
+
+  test('should return zero seconds', () => {
+    expect(getDuration(NOW, NOW)).toBe('0s');
+  });
+
+  test('should contain seconds', () => {
+    expect(getDuration(NOW, NOW + ONE_SECOND * 20)).toMatch(/[0-9]+s+/);
```

Comment:
I think it would be better to use strings instead of regular expressions, so anyone can look at tests and understand what this function can return in which case.